lib/screen/profil_screen.dart

```diff
@@ -0,0 +1,193 @@
+              style: TextStyle(
+                  fontSize: 25,
+                  fontWeight: FontWeight.bold,
+                  color: Theme.of(context).colorScheme.onSurface))),
+      body: BlocProvider(
+        create: (context) => SubscriptionBloc()..add(LoadSubscription()),
+        child: BlocBuilder<SubscriptionBloc, SubscriptionState>(
+          builder: (context, state) {
+            if (state is SubscriptionLoading) {
+              return const Center(child: CircularProgressIndicator());
+            }
+
+            if (state is SubscriptionLoaded) {
+              String subscriptionEndDate = state.isSubscribed &&
+                      state.subscription != null
+                  ? DateFormat('dd/MM/yyyy').format(state.subscription!.endDate)
+                  : "";
+
+              return FutureBuilder<CompleteUser?>(
+                future: UserService.getCompleteUserByUid(
+                    AuthService.currentUser!.uid),
+                builder: (BuildContext context,
+                    AsyncSnapshot<CompleteUser?> completeUser) {
+                  if (completeUser.connectionState == ConnectionState.waiting) {
+                    return const Center(child: CircularProgressIndicator());
+                  } else if (completeUser.hasError) {
+                    return Center(child: Text('Error: ${completeUser.error}'));
+                  } else if (completeUser.hasData) {
+                    var data = completeUser.data!;
+                    DateTime birthDateAsDate = data.birthDate.toDate();
+                    String formattedBirthDate =
+                        DateFormat('dd/MM/yyyy').format(birthDateAsDate);
+                    return ListView(
+                      shrinkWrap: true,
+                      children: [
+                        _buildTextItem(context, "Prénom", data.firstName),
+                        const SizedBox(height: 20),
+                        _buildTextItem(context, "Nom", data.lastName),
+                        const SizedBox(height: 20),
+                        _buildTextItem(context, "Email", data.email),
+                        const SizedBox(height: 20),
+                        _buildTextItem(
+                            context, "Date de naissance", formattedBirthDate),
+                        const SizedBox(height: 20),
+                        _buildTextItem(context, "Code postal", data.zipCode),
+                        const SizedBox(height: 20),
+                        if (subscriptionEndDate.isNotEmpty)
+                          _buildTextItem(
+                              context, "Fin d'abonnement", subscriptionEndDate),
+                        const SizedBox(height: 15),
+                        if (subscriptionEndDate.isEmpty)
+                          Align(
+                            alignment: Alignment.center,
+                            child: SizedBox(
+                              width: 325,
```

Comment:
Ne pas defenir la largeur directement comme ça, un petit telephone ne pourra pas tout voir. utilise MediaQuery.of(context).size.width * 0.9. c'est ce qu'on a utilisé partout dans l'app